docs/source/recipes/creating-a-custom-runtime-image.md

```diff
@@ -22,15 +22,16 @@ A runtime image provides the execution environment in which nodes are executed w
 
 Should none of these images meet your needs, you can utilize a custom container image, as long as it meets the following pre-requisites:
 
-- The container image is published on a public container registry (e.g. [https://hub.docker.com](https://hub.docker.com)). (Elyra currently does not support private registries.)
+- The image is stored in a container registry in a public or private network that the container platform in which the pipeline is executed can connect to. Examples of such registries are hub.docker.com or a self-managed registry in an intranet environment.
+- The image can be pulled from the registry without the need to authenticate. 
 - [Python 3](https://www.python.org/) is pre-installed and in the search path. Python versions that have reached their "end of life" are not supported.
 - [`curl`](https://curl.haxx.se/) is pre-installed and in the search path.
 
 Refer to the [Additional considerations](#additional-considerations) section for important implementation details.
 
 ## Requirements
 
-To create a custom container image for [https://hub.docker.com](https://hub.docker.com)) you need
+To create a custom container image and publish it on [https://hub.docker.com](https://hub.docker.com)) you need
```

Comment:
I think this reads better if it's just the website rather than a URL.
```suggestion
To create a custom container image and publish it on [hub.docker.com](https://hub.docker.com)) you need
```
Also, it looks like an extraneous `)` that is not rendered, but extraneous.